test/resources/layout-shared.css

```diff
@@ -12,6 +12,12 @@ body {
     padding: 20px;
 }
 
+.input-group-button-playground, .input-group-playground {
+  font-size: 14px;
+  padding: 0;
+  margin: 0;
+  border: none;
+}
```

Comment:
I know these PRs crossed in the night, but hopefully the input-group branch will help reduce the need for this.